alfred-telemetry-solr/README.md

```diff
@@ -105,77 +105,77 @@ At the moment following metrics are included, for each core tracked by solr:
-| solr_alfresco__afts_5minRateRequestsPerSecond                         |
-| solr_alfresco__afts_15minRateRequestsPerSecond                        |
-| solr_alfresco__afts_avgRequestsPerSecond                              |
-| solr_alfresco__cmis_handlerStart                                      |
-| solr_alfresco__cmis_requests                                          |
-| solr_alfresco__cmis_timeouts                                          |
-| solr_alfresco__cmis_errors                                            |
-| solr_alfresco__cmis_clientErrors                                      |
-| solr_alfresco__cmis_serverErrors                                      |
-| solr_alfresco__cmis_75thPcRequestTime                                 |
-| solr_alfresco__cmis_95thPcRequestTime                                 |
-| solr_alfresco__cmis_999thPcRequestTime                                |
-| solr_alfresco__cmis_medianRequestTime                                 |
-| solr_alfresco__afts_avgTimePerRequest                                 |
-| solr_alfresco__cmis_totalTime                                         |
-| solr_alfresco__cmis_5minRateRequestsPerSecond                         |
-| solr_alfresco__cmis_15minRateRequestsPerSecond                        |
-| solr_alfresco__cmis_avgRequestsPerSecond                              |
-| solr_alfresco_searcher_warmupTime                                     |
-| solr_alfresco_searcher_deletedDocs                                    |
-| solr_alfresco_searcher_maxDoc                                         |
-| solr_alfresco_searcher_numDocs                                        |
-| solr_alfresco_searcher_indexVersion                                   |
+| solr.alfresco./afts.handlerStart                                      |
+| solr.alfresco./afts.requests                                          |
+| solr.alfresco./afts.timeouts                                          |
+| solr.alfresco./afts.errors                                            |
+| solr.alfresco./afts.clientErrors                                      |
+| solr.alfresco./afts.serverErrors                                      |
+| solr.alfresco./afts.75thPcRequestTime                                 |
+| solr.alfresco./afts.95thPcRequestTime                                 |
+| solr.alfresco./afts.99thPcRequestTime                                 |
+| solr.alfresco./afts.999thPcRequestTime                                |
+| solr.alfresco./afts.medianRequestTime                                 |
+| solr.alfresco./afts.avgTimePerRequest                                 |
+| solr.alfresco./afts.totalTime                                         |
+| solr.alfresco./afts.5minRateRequestsPerSecond                         |
+| solr.alfresco./afts.15minRateRequestsPerSecond                        |
+| solr.alfresco./afts.avgRequestsPerSecond                              |
+| solr.alfresco./cmis.handlerStart                                      |
```

Comment:
would it make sense to make `afts` and `cmis` just tags ? they have all the same metrics etc - and could be added together, right ?

Reply:
Makes sense sematically, it's custom code.